express-sendfile/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('./node_modules/express');
+
+const app = express();
+
+app.use(function (req, res, next) {
+  const options = {
+    root: __dirname
+  };
+  res.sendFile('./hello.txt', options, function (err) {
+    if (err) {
+      next(err);
+    }
+  });
+  res.sendFile('./index.html', options, function (err) {
+    if (err) {
+      next(err);
+    }
+  });
+  res.sendFile('./styles.css', options, function (err) {
+    if (err) {
+      next(err);
+    }
+  });
+  res.sendFile('./main.js', options, function (err) {
```

Comment:
The server can't respond more than once, so you're going to need some conditional logic to send a different file (based on `req.originalUrl`)